docs/configuration.md

```diff
@@ -130,9 +130,11 @@ must match all alerts (i.e. not have any configured matchers).
 It then traverses the child nodes. If `continue` is set to false, it stops
 after the first matching child. If `continue` is true on a matching node, the
 alert will continue matching against subsequent siblings.
+If an alert matches a child node with `continue` set to true, even if it is the
+last sub-route, it will not trigger the default route. 
```

Comment:
This behavior is correct but it is the same as with any other routes. I think we should not add this wording, because this is what happens with any other route. We should not let readers believe that the default route is "special". (or do we have a bug?)